openshift/templates/images/nodejs/Dockerfile

```diff
@@ -13,12 +13,12 @@ LABEL io.k8s.description="Platform for building Node.js Applications" \
 # to make it clear why the following COPY operation is happening
 LABEL io.openshift.s2i.scripts-url=image:///usr/local/s2i
 
-# Copy the S2I scripts from ./.s2i/bin/ to /usr/local/s2i when making the builder image
-#COPY ./s2i/bin/ /usr/local/s2i
-
-# Copy extra files to the image.
+# Copy S2I base files to the image.
 COPY ./s2i/root/ /
-COPY ./s2i/api/ /
+
+# Copy the S2I scripts from ./s2i/api/ to /usr/local/s2i for the API runtime image.
+# This runs the migrations and seeds, then runs the server.
+COPY ./s2i/api/ /usr/local/s2i
```

Comment:
Oh I see it now.

this way the api can live in /user/local/s2i, and the app can live in /

Reply:
not exactly... the app doesn't need to live anywhere, as it is only used for the build, so all we need is node to assist in the build, after that this container is thrown away.

Reply:
If that is the case, why do we need two copies of it?